pkg/controller/certs/certs.go

```diff
@@ -48,12 +50,34 @@ func (kp *KeyPair) ToPEM() (certPEM []byte, privPEM []byte, err error) {
 	return
 }
 
+// ToTLSSecret converts the KeyPair to a Kubernetes TLS Secret
+func (kp *KeyPair) ToTLSSecret(name, namespace string) (*corev1.Secret, error) {
```

Comment:
I can't see where these are used?

Reply:
Testing. I can move these to the appropriate test files.